ios/FluentUI/Bottom Sheet/BottomSheetView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class BottomSheetView: UIView {
```

Comment:
what about the dimmingView?

Reply:
Dimming is usually used for modal sheets and isn't in specs for the persistent sheet as far as I'm aware. With this one we want the main content to be interactable even when the sheet is expanded, because often that will just be 1/3rd, 1/2 of the screen. I'm happy to add it later if there's a need.

Reply:
I do see dimming view in the design spec and did get confirmation from designers before.

Reply:
Ah I see it now. I need to talk to them more about how it should show, because I don't think it should in non-modal scenarios. The only case where it sort of makes sense is when it expands to full screen.

Reply:
Talked to design and it looks like it’s still an open question. It’s likely it will be needed in some form, but I’ll add it in a future PR once I get confirmation if that’s okay. There’s some details that need to worked out, like does it only start showing if we’re expanding into the upper third, etc.